modules/nodes-scheduler-taints-tolerations-about.adoc

```diff
@@ -238,6 +238,11 @@ To ensure backward compatibility, the daemon set controller automatically adds t
 
 You can also add arbitrary tolerations to daemon sets.
 
+[NOTE]
+====
+The control plane also adds the `node.kubernetes.io/memory-pressure` toleration on pods that have a QoS class. This is because Kubernetes manages pods in the `Guaranteed` or `Burstable` QoS classes. The new BestEffort pods do not get scheduled onto the affected node.
```

Comment:
Seems "BestEffort" should have some sort of formatting (**bold** or `monospace`?), or be changed to "best effort". If it is a QoS class like `Guaranteed` or `Burstable`, probably `monospace`.

Reply:
Addressed